docs/reference/remap/functions/parse_query_string.cue

```diff
@@ -0,0 +1,38 @@
+package metadata
+
+remap: functions: parse_query_string: {
+	category: "Parse"
+	description: #"""
+		Parses the `value` as a query string.
+		"""#
+	notices: [
+		"""
+			All values are returned as strings. We recommend manually coercing values to desired types as you see fit. Empty keys and values are allowed.
+			""",
+	]
+
+	arguments: [
+		{
+			name:        "value"
+			description: "The string to parse."
+			required:    true
+			type: ["string"]
+		},
+	]
+	internal_failure_reasons: []
+	return: types: ["object"]
+
+	examples: [
+		{
+			title: "Parse query string"
+			source: #"""
+				parse_query_string("foo=%2B1&bar=2&bar=3&xyz")
```

Comment:
Ruby on Rails (😱 @binarylogic) uses (used?) query strings like this for arrays: `?ids[]=1025&ids[]=1027`. Should we special-case that to group them into an array of `ids` similar to how we group `bar` in this example?

Alternatively, we keep things as they are, but add a (tested) example to the documentation that shows how this works.

Reply:
Added an example/test - we'd parse it as `"ids[]" = ["1025", "1027"]`. Since it's a RoR-specific way of handling array parameters I would keep it simple. WDYT?